tests/acceptance/link-test.js

```diff
@@ -1,7 +1,7 @@
-import { describe, it, beforeEach, afterEach } from 'mocha'
-import { expect } from 'chai'
-import startApp from '../helpers/start-app'
 import destroyApp from '../helpers/destroy-app'
+import startApp from '../helpers/start-app'
+import { expect } from 'chai'
```

Comment:
If we all agree then we can simply add http://eslint.org/docs/rules/object-curly-spacing to eslint-config-frost-standard.